handout/architect/architect-guide-source-code-to-working-software.md

```diff
@@ -0,0 +1,128 @@
+# Angular Source Code To Working Software
+
+_Scope Range_
+
+```
+    * Out of Scope *
+                                                    ^
+             Hardware                               |
+              Kernel                                | Possible passing reference 
+             Userland                               |
+            ----------                              v
+    JavaScript Runtime and/or DOM                   ^
+             ES5 Code                               |
+                                     TypeScript     | Layers referenced in 
+                                       Angular      | outline 
+                                       App Code     |
+                                                    v
+    * In Scope *
+    
+```
+
+## Act I - The Scene
+
+### Introduction
+
+- Describe the stack at an abstract level.  
+    - Possibly take a top down approach noting one aspect of the translation
+      moving down to the next layer
+    - Possibly take a bottom up approach while noting one thing that created a 
+      need for a new layer of abstraction 
```

Comment:
I prefer top down - that will take the reader from what they think they know to what they almost certainly don't.